mods/parsoid.js

```diff
@@ -618,7 +614,7 @@ PSP.makeTransform = function(from, to) {
             var innerRes = res.body[to];
```

Comment:
The bug is around here: for `to === 'wikitext'` Parsoid returns an unwrapped response.  From what I can tell from testing, this should look like

```
var innerRes = {};
if (to === 'wikitext') {
    innerRes.body = res.body;
    innerRes.headers = {
        'content-type': res.headers['content-type']
    }
} else {
    innerRes = res.body[to];
}
```
And also the content-type header is different from what we expect, the charset is the second param, not the third.